app/views/offshore/TaxBeforeFiveYearsView.scala.html

```diff
@@ -0,0 +1,69 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.hmrcfrontend.views.html.components.CharacterCount
+@import viewmodels.InputWidth._
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    hmrcCharacterCount: HmrcCharacterCount,
+    govukInput: GovukInput,
+    govukButton: GovukButton,
+    paragraph: components.paragraph,
+    link: components.link
+)
+
+@(form: Form[_], mode: Mode, year: String)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("taxBeforeFiveYears.title", year))) {
+
+    @formHelper(action = controllers.offshore.routes.TaxBeforeFiveYearsController.onSubmit(mode), 'autoComplete -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
+        <h1 class="govuk-heading-xl">@messages("taxBeforeFiveYears.heading", year)</h1>
+
+        @hmrcCharacterCount(
+            CharacterCount(
+                id = "taxBeforeFiveYears",
+                name = "taxBeforeFiveYears",
+                maxLength = Some(500), 
```

Comment:
I think the max length needs to be 5000, what does the ticket say?

Reply:
Fixed!!